tests/unit/test_docker_container.py

```diff
@@ -19,3 +19,56 @@ def test_dr_command_class():
     # test whether mutable params are not mutable across instances
     simple.options += ["spy"]
     assert "spy" not in DockerRunCommand().options
+
+
+def test_mapped_port_open():
+    command = DockerRunCommand(additional_opts=["-p", "80:80"])
+    image = DockerImage('nginx')
```

Comment:
could you please use `python3 -m http.server` instead? so we depend on as few images as possible:

https://github.com/fedora-modularity/conu/blob/master/docs/source/examples/webserver.py#L39